docs/openshift/logging-in/web-console-overview.md

```diff
@@ -1,7 +1,8 @@
 # Web Console Overview
 
-The NERC's OpenShift Container Platform web console a web-based user interface for
-performing common management tasks such as building and deploying applications.
+The NERC's OpenShift Container Platform web console a web-based user interface
```

Comment:
The NERC's OpenShift Container Platform (OCP) has a web-based console that can be used to perform common management tasks such as building and deploying applications:

[https://console.apps.shift.nerc.mghpcc.org](https://console.apps.shift.nerc.mghpcc.org)

The web console provides tools to access and manage your application code and data.

Reply:
I rephrase it as:

The NERC's OpenShift Container Platform (OCP) has a web-based console that can be
used to perform common management tasks such as building and deploying applications.

 You can find it at [https://console.apps.shift.nerc.mghpcc.org/](https://console.apps.shift.nerc.mghpcc.org/).